tests/test_aea.py

```diff
@@ -126,7 +138,10 @@ def test_react():
         message_bytes = DefaultSerializer().encode(msg)
 
         envelope = Envelope(
-            to=address, sender=address, protocol_id="default", message=message_bytes
+            to=address,
+            sender=address,
+            protocol_id="fetchai/default:0.1.0",
```

Comment:
notice, `protocol_id` is converted in the `Envelope` constructor`

Reply:
yeah I don't like that..